internal/controller/operatorconfigmap_controller.go

```diff
@@ -620,16 +617,30 @@ func (c *OperatorConfigMapReconciler) reconcileDelegatedCSI(storageClients *v1al
 	enableCephFsDriver := c.shouldEnableDriver(enableCephFsDriverKey)
 	enableNfsDriver := c.shouldEnableDriver(enableNfsDriverKey)
 
+	var useHostNetForRbdCtrlPlugin, useHostNetForCephFsCtrlPlugin, useHostNetForNfsCtrlPlugin bool
+
 	// if the storage client status has the driver requirements info, then it has higher precedence than the configmap.
 	for i := range storageClients.Items {
 		if storageClients.Items[i].Status.RbdDriverRequirements != nil {
 			enableRbdDriver = true
+			if storageClients.Items[i].Status.RbdDriverRequirements.CtrlPluginHostNetwork != nil &&
+				*storageClients.Items[i].Status.RbdDriverRequirements.CtrlPluginHostNetwork {
+				useHostNetForRbdCtrlPlugin = ptr.Deref(storageClients.Items[i].Status.RbdDriverRequirements.CtrlPluginHostNetwork, false)
+			}
```

Comment:
Can this be simplified to this?
```suggestion
			if useHostNetwork:=storageClients.Items[i].Status.RbdDriverRequirements.CtrlPluginHostNetwork; useHostNetwork != nil {
				useHostNetForRbdCtrlPlugin = ptr.Deref(useHostNetwork, false)
			}
```